projects/aca-content/src/lib/components/sidenav/navigation-menu/navigation-menu.component.html

```diff
@@ -0,0 +1,35 @@
+<button mat-icon-button [matMenuTriggerFor]="menu" aria-label="matMenu" class="app-menu-button">
+    <mat-icon svgIcon="menu" title="{{'APP.TOOLTIPS.OPTIONS_SETTINGS' | translate}}"></mat-icon>
```

Comment:
this makes zero sense.... the menu says "A" letter, what does that mean? the initials of the Admin? why every user should get static "A" image?